config/manager/kustomization.yaml

```diff
@@ -1,2 +1,8 @@
 resources:
-  - manager.yaml
+- manager.yaml
```

Comment:
this file shouldn't be changed in this PR

Reply:
Yes. I don't understand what happens locally. I keep file from main branch of principal repository.
Apologize.

Reply:
It don't know why but it seems that something in prepare-e2e (or e2e) targets, something modify this file.
I didn't commit it again.